test/python/compiler/test_transpiler.py

```diff
@@ -1257,22 +1261,23 @@ def test_multiqubit_gates_calibrations(self, opt_level):
         custom_gate = Gate("my_custom_gate", 5, [])
         circ.append(custom_gate, [0, 1, 2, 3, 4])
         circ.measure_all()
-        backend = FakeBoeblingen()
-        with pulse.build(backend, name="custom") as my_schedule:
+        backend = FakeGeneric(num_qubits=6)
+
+        with pulse.build(backend=backend, name="custom") as my_schedule:
             pulse.play(
-                pulse.library.Gaussian(duration=128, amp=0.1, sigma=16), pulse.drive_channel(0)
+                pulse.library.Gaussian(duration=128, amp=0.1, sigma=16), pulse.DriveChannel(0)
             )
```

Comment:
Why do all these need to change?

Reply:
@jakelishman , This is my silly mistake, I remember back in May or June, at that time I didn't implement drive_channel method and pulse capabilities to FakeGeneric(), so did some changes here but forgot to revert back.